src/haddock/modules/analysis/ilrmsdmatrix/__init__.py

```diff
@@ -53,8 +52,17 @@ def __init__(self, order, path, *ignore, init_params=DEFAULT_CONFIG,
         super().__init__(order, path, init_params)
 
     @classmethod
-    def confirm_installation(cls):
-        """Confirm if contact executable is compiled."""
+    def confirm_installation(cls) -> None:
+        """Confirm if FCC is installed and available."""
+        dcfg = read_from_yaml_config(DEFAULT_CONFIG)
+        exec_path = Path(RMSD_path, dcfg["executable"])
+
+        if not os.access(exec_path, mode=os.F_OK):
+            raise Exception(f"Required {str(exec_path)} file does not exist.")
+
+        if not os.access(exec_path, mode=os.X_OK):
+            raise Exception(f"Required {str(exec_path)} file is not executable")
```

Comment:
Didn't @amjjbonvin say that if the binary is not available it should fallback to the python subroutine?